CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
-(None)
+- Use the appropriate HTTP verbs for invokes and avoid resetting project IAM policy on deletion [#175](https://github.com/pulumi/pulumi-google-native/pull/175)
```

Comment:
Let's split this into two bullet points?

Reply:
Done.